rio_tiler/models.py

```diff
@@ -65,7 +65,7 @@ class Info(SpatialInfo):
     class Config:
         """Config for model."""
 
-        extra = "ignore"
+        extra = "allow"
```

Comment:
width, height, overviews are not set in the model because for `mosaic` or `stac` it doesn't really make sense. It's easier to just `allow` extra value in the model